server/services/workAndSkillsPageService.test.ts

```diff
@@ -26,38 +24,43 @@ import { pagedActiveAlertsMock } from '../data/localMockData/pagedAlertsMock'
 import { prisonApiClientMock } from '../../tests/mocks/prisonApiClientMock'
 
 describe('WorkAndSkillsService', () => {
-  describe('Scenario A', () => {
-    let prisonApiClient: PrisonApiClient
+  const prisonApiClient = prisonApiClientMock()
```

Comment:
Quite a bit of refactoring in this service class, but its was all about removing some duplication of the mock setup between "Scenario A" and "Scenario B" (whatever they are! I did not name those scenarios! )